src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -442,6 +448,11 @@ uint32_t PolicyHandler::GetAppIdForSending() const {
       logger_,
       "Number of apps with NONE level: " << apps_with_none_level.size());
 
+  if (apps_with_none_level.empty() && apps_without_none_level.empty()) {
```

Comment:
@IGapchuk I think check `apps_without_none_level` is redundant as it is handled at the beginning of this fuinction

Reply:
@AKalinich-Luxoft Has been removed in 0b50b09dccb77273e6047d277a99ef2ad28baaac